Dockerfile

```diff
@@ -36,7 +36,7 @@ COPY --from=build-hapi --chown=1001:1001 /tmp/hapi-fhir-jpaserver-starter/opente
 ENV ALLOW_EMPTY_PASSWORD=yes
 
 ########### distroless brings focus on security and runs on plain spring boot - this is the default image
-FROM gcr.io/distroless/java17-debian11:nonroot as default
+FROM gcr.io/distroless/java17-debian11:nonroot-arm64 as default
```

Comment:
Is this intentional? I think arm64 is stil the exception and not the rule...

Reply:
Sorry this was meant to be a local MR, I'll remove this change for the real MR.

Reply:
I'll close this and open another

Reply:
Opened here https://github.com/hapifhir/hapi-fhir-jpaserver-starter/pull/500